mscore/importptb.h

```diff
@@ -242,7 +242,7 @@ class PowerTab {
             struct ptTrack;
             struct ptSection {
                   int                                 number{ 0 };
-                  int                                 staffs{ 0 };
+                  int                                 staves{ 0 };
```

Comment:
Regarding this (and similar cases), I don't think this many spaces between are needed. Try deleting some.

Reply:
The whole file is (hum hum) specially indented. I am not sure I want to get into that...

Reply:
Suit yourself. I just get uncomfortable seeing these "special indentations".